src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -0,0 +1,48 @@
+package bit.javaoop;
+
+import java.util.List;
+
+public class FixedBudgetManager extends Manager {
+    private double budget;
+
+    public FixedBudgetManager(int salary, String name, double budget) {
+        super(salary, name);
+        this.budget = budget;
+    }
+
+    @Override
+    public boolean isSatisfied() {
+        return (this.budget < 15_000 && this.getSalary() > 20_000);
```

Comment:
- Apropo słówka [this](https://docs.google.com/document/d/1o0sQmxtOjMXZAtZOXV5HDTAD4oGtsCV-lnSJ18EUuA8/edit#bookmark=id.5x8o0h8jch5l)
- Przy wieloktornych warunkach w ifach itp. warto ponawiasować niekoniecznie wszystko co konkretne warunki w ten sposób: `(this.budget < 15_000) && (this.getSalary() > 20_000)`
- Znów _magic numbers_